config/clients/java/template/.github/workflows/main.yml.mustache

```diff
@@ -0,0 +1,27 @@
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ main, master ]
```

Comment:
we are not using master, so we can remove master from the list.

Reply:
Oops, I didn't notice this. Will remove

Reply:
Removed in [2080d74](https://github.com/openfga/sdk-generator/pull/160/commits/2080d74e72d5b19f242c1bcaee3585a9fc93fd00)